### ISSUE TITLE
Create a StatisticRecorder

### ISSUE THREAD

**Issue by hferentschik** [fabric8-services/fabric8-jenkins-proxy#115]
The idea is that the _StatisticRecorder_ is a component running in a separate goroutine listening on a stats channel and updating the stats in the DB on each channel read.

This relates to issue #75

**Comment by aslakknutsen** [comment]
@hferentschik Statistic as in Metrics? If so why not just expose it as Prometheus data?

**Comment by hferentschik** [comment]
No, statistics in terms of users accessing the Jenkins service. The proxy writes some stats data into a Postgres database.
